RileyLink/DeviceDataManager.swift

```diff
@@ -132,3 +132,15 @@ extension DeviceDataManager: DeviceManagerDelegate {
 
     func deviceManager(_ manager: DeviceManager, logEventForDeviceIdentifier deviceIdentifier: String?, type: DeviceLogEntryType, message: String, completion: ((Error?) -> Void)?) {}
 }
+
+// MARK: - DeviceAlertPresenter
+extension DeviceDataManager: DeviceAlertPresenter {
+    func issueAlert(_ alert: DeviceAlert) {
```

Comment:
@ps2 I wasn't aware that there was another `DeviceManagerDelegate` implementor in `RileyLink`...is this a separate app?
I'm a bit wary of making these no-ops, because it would mean that the 'new' alert system will not work for this app.  What say you?

Reply:
This app is not used in any production sense, so it's ok.